api/migrations/versions/4923563b3a63_add_reponseID_outcome.py

```diff
@@ -0,0 +1,42 @@
+""" Add foreign key response_id to outcome
+
+Revision ID: 4923563b3a63
+Revises: bf4cb562e36c
+Create Date: 2024-12-07 19:07:18.021334
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4923563b3a63'
+down_revision = 'bf4cb562e36c'
+
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column(
```

Comment:
This will break when we try to run it on the production database because there are already many records in the table, so as soon as it tries to add the column which doesn't allow nulls, it will fail. 
So what you'll need to do is:
- Add the column response_id with nullable=True
- Update the existing data to set the correct response_id
- Update the column to set nullable=False
Here's an example of a migration that does something similar: https://github.com/deep-learning-indaba/Baobab/blob/8fb7327c8976bcd54d278804a321c7609811cca5/api/migrations/versions/e8d1ee2746e0_event_travel_grant.py#L19